api_spec.rb

```diff
@@ -81,21 +81,21 @@ def check_dir_listing_content_type(content_type)
     end
   end
 
-  describe "PUT another JSON object" do
+  describe "updating that JSON object" do
     before do
       @old_outer_listing_res = do_get_request("#{CONFIG[:category]}/")
-      @old_listing_res = do_get_request("#{CONFIG[:category]}/some-other-subdir/")
-      @res = do_put_request("#{CONFIG[:category]}/some-other-subdir/test-object-simple.json",
-                            '{"new": "object"}',
+      @old_listing_res = do_get_request("#{CONFIG[:category]}/some-subdir/")
+      @res = do_put_request("#{CONFIG[:category]}/some-subdir/nested-folder-object.json",
+                            '{"foo": "bam"}',
                             { content_type: "application/json" })
       @outer_listing_res = do_get_request("#{CONFIG[:category]}/")
-      @listing_res = do_get_request("#{CONFIG[:category]}/some-other-subdir/")
+      @listing_res = do_get_request("#{CONFIG[:category]}/some-subdir/")
 
       @item_etag = @res.headers[:etag]
       @old_listing = JSON.parse @old_listing_res.body
       @listing = JSON.parse @listing_res.body
-      @item_info = @listing["items"]["test-object-simple.json"]
-      @old_item_info = @old_listing["items"]["test-object-simple.json"]
+      @item_info = @listing["items"]["nested-folder-object.json"]
```

Comment:
I had to look it up because I'm usually using RSpec, there's an extension that adds `before :all` to minitest-spec (https://github.com/jeremyevans/minitest-hooks). We have two options: add this as a dependency or turn those three `it` blocks into one